saos-webapp/src/main/java/pl/edu/icm/saos/webapp/WebappConfiguration.java

```diff
@@ -67,6 +70,8 @@ public TilesConfigurer tilesConfigurer() {
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
         registry.addResourceHandler("/static/**").addResourceLocations("/WEB-INF/static/").setCachePeriod(3600*24*7);
         registry.addResourceHandler("/robots.txt").addResourceLocations("/WEB-INF/").setCachePeriod(0);
+        registry.addResourceHandler("/files/judgments/**").addResourceLocations(ResourceUtils.FILE_URL_PREFIX + judgmentsContentPath).setCachePeriod(3600*24);
```

Comment:
I would give much much smaller cache (for example 10minutes). Consecutive reading of the same file will be rare and will probably be done in short time period.